packages/fs-detectors/src/detect-instrumentation.ts

```diff
@@ -0,0 +1,70 @@
+import { DetectorFilesystem } from './detectors/filesystem';
+import semver from 'semver';
+
+interface InstrumentationDependency {
+  name: string;
+  minVersion: string;
+}
+
+const VERCEL_OTEL = '@vercel/otel';
+const OPENTELEMETRY_SDK_TRACE_NODE = '@opentelemetry/sdk-trace-node';
+const OPENTELEMETRY_API = '@opentelemetry/api';
+
+const tracingDependencies: InstrumentationDependency[] = [
+  { name: VERCEL_OTEL, minVersion: '1.11.0' },
+  { name: OPENTELEMETRY_SDK_TRACE_NODE, minVersion: '1.19.0' },
+  { name: OPENTELEMETRY_API, minVersion: '1.7.0' },
```

Comment:
`@opentelemetry/api` is not a very good indicator since this is just an API and not an instrumentation. Can we remove it?